dev/TabView/TabView.cpp

```diff
@@ -678,12 +686,9 @@ void TabView::OnItemsChanged(winrt::IInspectable const& item)
                 }
 
             }
-            // Last item removed, update sizes
-            // The index of the last element is "Size() - 1", but in TabItems, it is already removed.
             if (TabWidthMode() == winrt::TabViewWidthMode::Equal)
             {
-                m_updateTabWidthOnPointerLeave = true;
-                if (args.Index() == TabItems().Size())
+                if (!m_pointerInTabstrip)
```

Comment:
I think we also want to resize the items if it was the last item that was removed, even if the pointer is in the tab strip.

Reply:
Good point, that is also the reason of the test failures (all TabView interaction tests ran fine on my machine).